inst/WORDLIST

```diff
@@ -12,4 +12,5 @@ http
 https
 tidyverse
 yaml
-magrittr
\ No newline at end of file
+magrittr
+Maelle
```

Comment:
but my name doesn't come up anywhere?

Reply:
It does, in the updated NEWS, and `R CMD check` complains if I don't have this in the word list.